domain/user/state/state.go

```diff
@@ -429,6 +429,51 @@ func (st *State) SetActivationKey(ctx context.Context, name string, activationKe
 	})
 }
 
+// GetActivationKey retrieves the activation key for the user with the supplied
+// user name. If the user does not exist an error that satisfies
+// usererrors.NotFound will be returned.
+func (st *State) GetActivationKey(ctx context.Context, name string) ([]byte, error) {
+	db, err := st.DB()
+	if err != nil {
+		return nil, errors.Annotate(err, "getting DB access")
+	}
+
+	uuidStmt, err := st.getActiveUUIDStmt()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	m := make(sqlair.M, 1)
```

Comment:
Could've just used this to stick the key in instead of creating another type.

Reply:
It's messy because of the `[]byte` setup, nil vs string vs []byte. So if I just made a type it fixes it for me.